### PR TITLE
Feature#2/rails initialize

### LINKED ISSUE #2: rails_initialize
# REQUIREMENTS
Intilialize the proyect with ruby on rails

### PR DESCRIPTION
**Pull request by javs1**
# Description
What:
 - Rails initialization of the app
 - Added postgress as a BD
 - Added bootstrap
 - Added database.yml to .gitignore

Why:
- We use postgress for the DB to make easier the deploy to Heroku when necesary
- We use bootstrap to facilitate all front-end realted work
- We added the database.yml to the .gitignore for security

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

File: EasyLearn/.gitignore [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/tmp/storage/*
+!/tmp/storage/
+!/tmp/storage/.keep
+
+/public/assets
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+/config/database.yml
```
Comment (JuanequeX) [inline EasyLearn/.gitignore:37]: Nice, but where is it your database.yml.example file?

**Review by xFerchoVJ — APPROVED** [review]

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Comment by JuanequeX** [comment]
Remember guys is important complete the information of your pull request on the fields like `Assignees, labels and milestone`

**Review by BrandonVG — APPROVED** [review]

**Review by YilKanda — COMMENTED** [review]
Database.yml.example file missing.

File: EasyLearn/.gitignore (hunk 1 above)
Comment (javs1) [reply EasyLearn/.gitignore:37]: I added the database.yml.example in a new commit

**Review by JuanequeX — APPROVED** [review]
LGTM

File: EasyLearn/.gitignore (hunk 1 above)
Comment (MauricioNC) [reply EasyLearn/.gitignore:37]: In the previous kata 'reloj checador', someone told us that we don't need include the database.yml, and told us that put it in the .gitignore file, for that reason we ignore the database.yml

Comment (MauricioNC) [reply EasyLearn/.gitignore:37]: ooh sorry, I did'nt see the previous comment

Comment (JuanequeX) [reply EasyLearn/.gitignore:37]: Yes @MauricioNC the database.yml is the file that contain the credentials of database and it's important that each one of your have a unique configuration on your local project, but into this repositorie we need to see a structure reference of this  file and it is the database.yml.example

**Pull request merged**